ReactiveXComponent/Configuration/XCConfiguration.cs

```diff
@@ -7,37 +8,106 @@ public class XCConfiguration : IXCConfiguration
     {
         private readonly XCApiConfigParser _parser;
 
-        public XCConfiguration(Parser.XCApiConfigParser parser)
+        public XCConfiguration(XCApiConfigParser parser)
         {
             _parser = parser;
         }
 
         public void Init(Stream xcApiStream)
         {
-            _parser.Parse(xcApiStream);
+            try
+            {
+                _parser.Parse(xcApiStream);
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Failed to init configuration", ex);
```

Comment:
try to use specific exceptions (instead of generic ones) such as InitConfigurationException...